bsuite/utils/gym_wrapper.py

```diff
@@ -93,3 +94,77 @@ def reward_range(self) -> Tuple[float, float]:
   def __getattr__(self, attr):
     """Delegate attribute access to underlying environment."""
     return getattr(self._env, attr)
+
+
+class ReverseGymWrapper(dm_env.Environment):
+    """A wrapper that converts an OpenAI gym environment to a dm_env.Environment"""
```

Comment:
Looks like you're using 4 spaces here. Are you able to switch to 2 to be consistent with the rest of the codebase?